output.go

```diff
@@ -776,7 +776,10 @@ func OutputsSyntacticalDepositAmount(protoParams ProtocolParameters, rentStructu
 
 		// check whether the amount in the return condition allows the receiver to fulfill the storage deposit for the return output
 		if storageDep := output.UnlockConditionSet().StorageDepositReturn(); storageDep != nil {
-			minStorageDepositForReturnOutput := rentStructure.MinStorageDepositForReturnOutput(storageDep.ReturnAddress)
+			minStorageDepositForReturnOutput, err := rentStructure.MinStorageDepositForReturnOutput(storageDep.ReturnAddress)
+			if err != nil {
+				return ierrors.Wrapf(err, "output index %d", index)
```

Comment:
maybe extend the error message a bit, because we will only know about an overflow/underflow, and the output index. but not that it was about storage deposit calculation.